Source/Widgets/Animation/Animation.js

```diff
@@ -698,6 +703,31 @@ define([
      * animation.applyThemeChanges();
      */
     Animation.prototype.applyThemeChanges = function() {
+        // Since we rely on computed styles for themeing, we can't actually
+        // do anything if the container has not yet been added to the DOM.
+        // Set up an observer to be notified when it is added and apply
+        // the changes at that time.
+        if (!document.body.contains(this._container)) {
+            if (defined(this._observer)) {
+                //Already listening.
+                return;
+            }
+            var that = this;
+            that._observer = new MutationObserver(function(mutations) {
+                mutations.forEach(function(mutation) {
+                    for (var i = 0; i < mutation.addedNodes.length; i++) {
+                        if (mutation.addedNodes[i].contains(that._container)) {
```

Comment:
Looking at the code, I can eliminate this `forEach` anyway and use a single `document.body.contains` check, so this should work for you now.  However, I would recommend making sure you have the latest IE on your system, perhaps you're missing some patches or something.